packages/conf-libssl/conf-libssl.3/opam

```diff
@@ -23,7 +23,7 @@ depexts: [
   ["openssl-devel"] {os-distribution = "ol"}
   ["openssl-devel"] {os-distribution = "fedora"}
   ["libopenssl-devel"] {os-family = "suse"}
-  ["libressl-dev"] {os-family = "alpine"}
+  ["openssl-dev"] {os-distribution = "alpine"}
```

Comment:
why does this go from os-family to os-distribution?

Reply:
`os-distribution` is used in [`conf-libssl.{1,2}`](https://github.com/ocaml/opam-repository/pull/18090/files)